Makefile

```diff
@@ -1,5 +1,7 @@
+# Y - in response to Would you like to add Bootstrap(yes/no)
+
 create:
-	rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
+	printf "Y\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
```

Comment:
@rafayet-monon as this response happened only on applying web variant, so how about we separate the `create` here to `create_web` and `create_api`

```
# Y - in response to Would you like to add Bootstrap(yes/no)
create_web:
  printf "Y\n" | rails new $(APP_NAME) -m ./template.rb -T --web ${OPTIONS}

create_api:
  rails new $(APP_NAME) -m ./template.rb -T --api ${OPTIONS}
```

Reply:
Added in [d534b0d](https://github.com/nimblehq/rails-templates/pull/239/commits/d534b0d6ffef858976ae1c54b9db07701fbaca71)